msg/optical_flow_raw.msg

```diff
@@ -0,0 +1,17 @@
+# Optical flow in XYZ body frame in SI units.
```

Comment:
Since this is the same message format, can you use a multi-topic, like https://github.com/PX4/Firmware/blob/57362d0987c7c9e7560b25931688cecaf0417413/msg/actuator_controls.msg#L16?

Reply:
sure